yara-x-dump/src/tests/testdata/macho_x86_file.out

```diff
@@ -0,0 +1,13 @@
+field1: 0x7b
+field2: "test"
+# Nested segments
```

Comment:
These `#Nested segments` and `#Nested optional` comments are not really necessary IMHO.